src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine[] machineArray = new Machine[3];
+
+        machineArray[0] = new Truck();
+        machineArray[1] = new Buldozer();
+        machineArray[2] = new Excavator();
+
+        for (Machine m : machineArray) {
+            m.doWork();
+            m.stopWork();
+        }
```

Comment:
```suggestion
        Machine[] machines = new Machine[3];
        machines[0] = new Truck();
        machines[1] = new Buldozer();
        machines[2] = new Excavator();
        for (Machine machine : machines) {
            m.doWork();
            m.stopWork();
        }
```

lets remove redundant empty lines and rename some variables

Reply:
Okay, i got it. Can you please accept this work, just to avoid another commit for fixing some empty lines)